src/app/checkout/step-1/step-1.component.js

```diff
@@ -67,6 +69,6 @@ export default angular
     controller: Step1Controller,
     templateUrl: template.name,
     bindings: {
-      changeStep: '&'
```

Comment:
It adds watchers... Not a big deal. I think you need to call it like [this](https://github.com/CruGlobal/give-web/pull/6/files#diff-723383532014b6b0bdf61e792b226b2cR64). 

So you'd use it like this:
```
<my-component change-step="$ctrl.parentComponentFunction(param1)"></my-component>
```
Parent component controller:
```
parentComponentFunction(someParam1){
  //do something
}
```
Child component:
```
this.changeStep({param1: 'value})
```

For some reason it uses named parameters so the key in the child component has to match what you put in the template. I guess it has to parse the function in the template as a string and then call it.